### PR TITLE
Update base image for Operator to v4.10

### PR DESCRIPTION
**Pull request by leifmadsen**
Depends on https://github.com/infrawatch/smart-gateway-operator/pull/124

### CODE REVIEW

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
Not really sure why this skipped the deployment (time out issues?), so gonna try running this again. If it fails again I'll have to do another local deployment and figure out why this is failing. I looked at the `tested_files` list and ran locally the `run_ci` check the `Jenkinsfile` does, and it seems like this is one of the files/locations that should result in testing happening.

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
One last time and if it fails I'll have to debug this next week.

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
Upon further local investigation, I found that the `meta` variable changed between versions 4.6 and 4.10 in the base image for Ansible Operators to `ansible_operator_meta`.

I did an inline replacement across the file with this script:

```
cd roles/servicetelemetry/
for i in $(grep -r "meta\." | sort | cut -d':' -f1 | uniq); do ^Cd -i 's/meta\./ansible_operator_meta\./g' $i; done
```

**Comment by csibbitt** [comment]
> Upon further local investigation, I found that the `meta` variable changed between versions 4.6 and 4.10 in the base image for Ansible Operators to `ansible_operator_meta`.
> 
> I did an inline replacement across the file with this script:
> 
> ```
> cd roles/servicetelemetry/
> for i in $(grep -r "meta\." | sort | cut -d':' -f1 | uniq); do ^Cd -i 's/meta\./ansible_operator_meta\./g' $i; done
> ```

I assume the command above is `sed -i 's/meta\./ansible_operator_meta\./g'` . I did some searches for potential other uses of meta, but I think this covers it.

**Review by csibbitt — APPROVED** [review]
lgtm

**Comment by leifmadsen** [comment]
> > Upon further local investigation, I found that the `meta` variable changed between versions 4.6 and 4.10 in the base image for Ansible Operators to `ansible_operator_meta`.
> > I did an inline replacement across the file with this script:
> > ```
> > cd roles/servicetelemetry/
> > for i in $(grep -r "meta\." | sort | cut -d':' -f1 | uniq); do ^Cd -i 's/meta\./ansible_operator_meta\./g' $i; done
> > ```
> 
> I assume the command above is `sed -i 's/meta\./ansible_operator_meta\./g'` . I did some searches for potential other uses of meta, but I think this covers it.

Oops! Bad copy/pasta from console. Here is the command ran:

```
for i in $(grep -r "meta\." | sort | cut -d':' -f1 | uniq); do sed -i 's/meta\./ansible_operator_meta\./g' $i; done
```

**Comment by leifmadsen** [comment]
I'm just merging this because it shouldn't be affecting the smoketests at all, and I've done a full reproduction with quicklab using the same process the Jenkinsfile provides, and all smoketests etc pass. I'll have to continue debugging this if we keep running into it, but I don't see how this is failing the deployment so late into the game, and with external scripts.

**Comment by leifmadsen** [comment]
🤠

**Comment by leifmadsen** [comment]
> I'm just merging this because it shouldn't be affecting the smoketests at all, and I've done a full reproduction with quicklab using the same process the Jenkinsfile provides, and all smoketests etc pass. I'll have to continue debugging this if we keep running into it, but I don't see how this is failing the deployment so late into the game, and with external scripts.

This is a lie. I mean the other PR... oops! Not merging this yet.

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
Ha! I figured it out. Didn't make the same `meta` variable change in SGO... and this is a same-named deployment. 🤦🏻

**Comment by leifmadsen** [comment]
test

**Comment by leifmadsen** [comment]
Huzzuh! It works!

File: roles/servicetelemetry/tasks/_local_signing_authority.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@
       apiVersion: cert-manager.io/v1
       kind: Issuer
       metadata:
-        name: '{{ meta.namespace }}-selfsigned'
-        namespace: '{{ meta.namespace }}'
+        name: '{{ ansible_operator_meta.namespace }}-selfsigned'
```
Comment (elfiesmelfie) [inline roles/servicetelemetry/tasks/_local_signing_authority.yml:8]: I'm assuming that this is a change due to the var being renamed in the newer operator version

**Review by elfiesmelfie — APPROVED** [review]
Looks good to me, assuming that the only update needed was the var name change and the operator version change.

**Pull request merged**